usr/src/uts/common/fs/zfs/sys/zap.h

```diff
@@ -87,22 +88,17 @@ extern "C" {
 
 /*
  * Specifies matching criteria for ZAP lookups.
```

Comment:
Is this trying to keep track of 2 things:
1. should we do UTF form normalization?
2. should we do case insensitivity (can also be thought of as "case normalization")?
If so, can we represent this with a bitfield with 2 entries:
1. MT_NORMALIZED_FORM
2. MT_CASE_INSENSITIVE (or MT_NORMALIZED_CASE)
If neither bit is set, then we are looking for an exact (strcmp()) match.